docs/install-upgrade/config.md

```diff
@@ -1,149 +1,33 @@
-
-``` {.yaml .annotation linenums="1"}
-spec:
-    config:
-      control:
-        defaultUser: # user 'core' on all control nodes
-          password: "hashhashhashhashhash" # password hash
-          authorizedKeys:
-            - "ssh-ed25519 SecREKeyJumblE"
-
-        fabric:
-          mode: spine-leaf # "spine-leaf" or "collapsed-core"
-
-          defaultSwitchUsers:
-            admin: # at least one user with name 'admin' and role 'admin'
-              role: admin
-              #password: "$5$8nAYPGcl4..." # password hash
-              #authorizedKeys: # optional SSH authorized keys
-              #  - "ssh-ed25519 AAAAC3Nza..."
-            op: # optional read-only user
-              role: operator
-              #password: "$5$8nAYPGcl4..." # password hash
-              #authorizedKeys: # optional SSH authorized keys
-              #  - "ssh-ed25519 AAAAC3Nza..."
-
-```
+## Complete Fab.yaml Example File
 
-Control node(s) user is always named `core`.
-
-The role of the user,`operator` is read-only access to `sonic-cli` command on the switches. In order to avoid conflicts, do not use the following usernames: `operator`,`hhagent`,`netops`.
-
-### NTP and DHCP
-The control node uses public ntp servers from cloudflare and google by default. The control node runs a dhcp server on the management network. See the [example file](#complete-example-file).
-
-## Control Node
-The control node is the host that manages all the switches, runs k3s, and serves images. This is the YAML document configure the control node:
-``` {.yaml .annotation linenums="1"}
-apiVersion: fabricator.githedgehog.com/v1beta1
-kind: ControlNode
-metadata:
-  name: control-1
-  namespace: fab
-spec:
-  bootstrap:
-   disk: "/dev/sda" # disk to install OS on, e.g. "sda" or "nvme0n1"
-  external:
-    interface: enp2s0 # interface for external
-    ip:	dhcp # IP address for external interface
-  management:
-    interface: enp2s1 # interface for management
-
-# Currently only one ControlNode is supported
-```
-The **management** interface is for the control node to manage the fabric switches, *not* end-user management of the control node. For end-user management of the control node specify the **external** interface name.
-
-### Forward switch metrics and logs
-
-There is an option to enable Grafana Alloy on all switches to forward metrics and logs to the configured targets using
-Prometheus Remote-Write API and Loki API. If those APIs are available from Control Node(s), but not from the switches,
-it's possible to enable HTTP Proxy on Control Node(s) that will be used by Grafana Alloy running on the switches to
-access the configured targets. It could be done by passing `--control-proxy=true` to `hhfab init`.
```

Comment:
Some of the info is removed, and not replace, such as here. I understand you want to remove references to `hhfab` in these sections, is it OK to just remove these? Simply double-checking.

Reply:
It is more about removing the references to control-proxy.